### PR TITLE
enhancements to Ldp po_instance derived table with comments

### PR DESCRIPTION
**Pull request by natalyapik**
```
All queries:
- [ ] Pull request is based on a new branch (not main)
- [ ] Pull request scope is not overly broad
- [ ] Release notes have been added or updated in CHANGES.md
- [ ] Query runs without errors
- [ ] Query output is correct
- [ ] Query logic is clear and well documented
- [ ] Query is readable and properly indented with spaces (not tabs)
- [ ] Table and column names are in all-lowercase
- [ ] Quotation marks are used only where necessary

Report queries:
- [ ] Query has complete user documentation
    - [ ] Purpose of report
    - [ ] Sample output
    - [ ] Query instructions

Derived tables:
- [ ] Query begins with user documentation in comment lines
- [ ] File name is listed in `runlist.txt` after dependencies
- [ ] All columns have indexes
- [ ] Table is vacuumed and analyzed
```

### CODE REVIEW

**Review by trapido — COMMENTED** [review]
All queries:
- [x] Pull request is based on a new branch (not main)
- [x] Pull request scope is not overly broad
- [x] Release notes have been added or updated in CHANGES.md
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary

Report queries:
- [ ] Query has complete user documentation
    - [ ] Purpose of report
    - [ ] Sample output
    - [ ] Query instructions

Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

Everything looks good, just a couple of typos:
Line 129 and line 147 - "weather" should be "whether"
Line 3 -"no source' - the second quote should be a double-quote

File: sql/derived_tables/po_instance.sql [hunk 1]
```diff
@@ -43,44 +48,106 @@ FROM
...
+
+COMMENT ON COLUMN po_instance.pol_instance_hrid IS 'A human readable number of the instance record this purchase order line is related to';
+
+COMMENT ON COLUMN po_instance.pol_holding_id IS 'UUID of the holdings this purchase order line is related to';
+
+COMMENT ON COLUMN po_instance.pol_location_id IS 'UUID of the location created for this purchase order line';
+
+COMMENT ON COLUMN po_instance.pol_location_name IS 'Name of the purchase order line location';
+
+COMMENT ON COLUMN po_instance.pol_location_source IS 'Weather location of the material is a location or permanent location of the purchase order line in inventory';
```
Comment (LauSchl) [inline sql/derived_tables/po_instance.sql:147]: weather --> wether

**Review by LauSchl — COMMENTED** [review]
All queries:
- [x] Pull request is based on a new branch (not main)
- [x] Pull request scope is not overly broad
- [x] Release notes have been added or updated in CHANGES.md
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary

Report queries:
- [ ] Query has complete user documentation
    - [ ] Purpose of report
    - [ ] Sample output
    - [ ] Query instructions

Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

Runs fine :) as @trapido  said: 2 typos where "weather" instead of  "wether"

**Review by trapido — DISMISSED** [review]
All queries:
- [x] Pull request is based on a new branch (not main)
- [x] Pull request scope is not overly broad
- [x] Release notes have been added or updated in CHANGES.md
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary

Report queries:
- [ ] Query has complete user documentation
    - [ ] Purpose of report
    - [ ] Sample output
    - [ ] Query instructions

Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

**Review by LauSchl — DISMISSED** [review]
All queries:
- [x] Pull request is based on a new branch (not main)
- [x] Pull request scope is not overly broad
- [x] Release notes have been added or updated in CHANGES.md
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary

Report queries:
- [ ] Query has complete user documentation
    - [ ] Purpose of report
    - [ ] Sample output
    - [ ] Query instructions

Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

Looks good :)

**Pull request merged**